assets/scss/app.scss

```diff
@@ -10,7 +10,7 @@ ul {
   @extend .list-bullet;
 }
 
-a {
+a:link {
```

Comment:
what does this do?

Reply:
The `:link` property applies only to links with an href. Initially the govuk styling for links was assigned to anything with an anchor tag but because some of the menu links are disabled until required fields are completed,  I changed it so that the styling is only applied to enabled links i.e.those with an href.

Reply:
Ace 👍